src/libcore/mem.rs

```diff
@@ -325,21 +420,20 @@ pub unsafe fn uninitialized<T>() -> T {
     intrinsics::uninit()
 }
 
-/// Swap the values at two mutable locations of the same type, without deinitializing or copying
-/// either one.
+/// Swaps the values at two mutable locations, without deinitializing either one.
 ///
-/// # Examples
+/// # Example
```

Comment:
To explain why: "Examples" is a target for doc parsers, more consistent, and if somebody adds another example it doesn't become wrong (think braces around single-statement if statements). A good reward for the sacrifice of grammaticality.